frontends/p4/fromv1.0/converters.h

```diff
@@ -107,8 +108,9 @@ class ExternConverter {
         return get(e)->convertExternType(s, e, name); }
     static const IR::Declaration_Instance *cvtExternInstance(ProgramStructure *s,
                 const IR::Declaration_Instance *di, cstring name,
-                IR::IndexedVector<IR::Declaration> *scope) {
-        return get(di)->convertExternInstance(s, di, name, scope); }
+                IR::IndexedVector<IR::Declaration> *local_scope,
+                IR::IndexedVector<IR::Node> *global_scope) {
```

Comment:
will `global_scope` always be the same as `s->declarations`?  If so we don't really need it as an extra argument.  If not, what is it?